core/src/BitArray.h

```diff
@@ -168,7 +168,7 @@ class BitArray
 
 	/**
 	* @param i bit to get
-	* @return true iff bit i is set
+	* @return true if bit i is set
```

Comment:
Admittedly a bit math-slangy but that is not a spelling mistake but means "if and only if". Maybe spelling out like that would be best, indeed. See also the other occurrences.